src/main/java/com/treasuredata/bigdam/log/Log.java

```diff
@@ -37,12 +37,12 @@
     private static final String DEFAULT_DEBUG_TAG = "bigdam.log.debug";
     private static final String DEFAULT_TRACE_TAG = "bigdam.log.trace";
 
-    private static final int SENTRY_LEVEL_THRESHOLD_NEVER = 5;
-    private static final int SENTRY_LEVEL_THRESHOLD_ERROR = 4;
-    private static final int SENTRY_LEVEL_THRESHOLD_WARN = 3;
-    private static final int SENTRY_LEVEL_THRESHOLD_INFO = 2;
-    private static final int SENTRY_LEVEL_THRESHOLD_DEBUG = 1;
-    private static final int SENTRY_LEVEL_THRESHOLD_TRACE = 0;
+    private static final int REMOTE_LEVEL_THRESHOLD_NEVER = 5;
```

Comment:
`REMOTE_` sounds a bit weird to me. How about just `LOG_LEVEL_THRESHOLD_NEVER`

Reply:
`LOG_LEVEL_THRESHOLD_*` is also a bit confusing because log level of local logger is managed by underlying layer. This levels are only for remote logging services (like Fluentd and Sentry).
How about `LOG_SERVICE_LEVEL_THRESHOLD_*`?

Reply:
👍